src/react-components/auth/SignInPage.js

```diff
@@ -48,6 +48,10 @@ function useSignIn() {
     },
     [auth]
   );
+    // Expose the login method for the SSO scripts (if they are loaded)
+    if (window.HUBS_SSO){
```

Comment:
Instead of cluttering window, why not create APP.sso. as an object in the App object (that's where they store all kinds of things they want visible to the JS context)

Reply:
Are you thinking of putting it in the state? The App object on `window` gets reset to the store several times throughout the application lifecycle, so I left it in the window to minimize the number of code changes required to add this capability. I can add a new state object and schema to the store, just want to confirm that is the right direction before proceeding down that path.
https://github.com/aelatgt/hubs/commit/3572d1f0238827261967301f8a8619d45c3c692e
https://github.com/aelatgt/hubs/blob/4fe49a17367cf17e533f0a00ba6dbc356a271072/src/signin.js

Reply:
Hmm, didn't realize the App object gets recreated.  That seems odd.  I'll have to look into that; I was planning on putting some data in there.

If that's the case, using a different place is fine; can you you put all the data within one object?

Reply:
So, the window.APP object has a "store" property which gets updated, but the window.APP object itself should be safe to add things onto for the duration of the session.

Reply:
There are multiple pages where an `window.APP` object is created in the entry script for that page; e.g., index.js, hub.js, signin.js. But the APP object is only created once for each page session.